mappings/net/minecraft/server/world/ServerWorld.mapping

```diff
@@ -22,6 +22,8 @@ CLASS net/minecraft/class_3218 net/minecraft/server/world/ServerWorld
 	FIELD field_26934 entityList Lnet/minecraft/class_5574;
 	FIELD field_26935 entityManager Lnet/minecraft/class_5579;
 	FIELD field_28859 sleepManager Lnet/minecraft/class_5838;
+	FIELD field_29768 ENTITY_TICK_FREQUENCY I
+		COMMENT The number of ticks the world will wait between entity updates.
```

Comment:
Hmm, so this is not `1`? Interesting, I thought entities are always ticked

Reply:
```suggestion
		COMMENT The number of ticks ({@value}) the world will wait between entity updates.
```
Javadoc will render the value on the web page with this little tag.

Reply:
@liach I thought so too, but evidently not. At least, I think I should change the description a little. Looks like it only applies when the server is idle.

```
        boolean bl2 = bl4 = !this.players.isEmpty() || !this.getForcedChunks().isEmpty();
        if (bl4) {
            this.resetIdleTimeout();
        }
        if (bl4 || this.idleTimeout++ < 300 /* ENTITY_TICK_FREQUENCY */) {
            lv.push("entities");
```